Sources/SnapshotPreviewsCore/AppKitRenderingStrategy.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  AppKitRenderingStrategy.swift
+//
+//
+//  Created by Noah Martin on 8/8/24.
+//
+
+#if canImport(AppKit)
+import Foundation
+import AppKit
+import SwiftUI
+import SnapshotSharedModels
+
+class BorderlessWindow: NSWindow {
+    override init(contentRect: NSRect, styleMask style: NSWindow.StyleMask, backing bufferingType: NSWindow.BackingStoreType, defer flag: Bool) {
+        super.init(contentRect: contentRect, styleMask: [.borderless], backing: bufferingType, defer: flag)
+        self.isOpaque = false
+        self.backgroundColor = NSColor.clear
+    }
+}
+
+public class AppKitRenderingStrategy: RenderingStrategy {
+
+  private let window: NSWindow
+
+  public init() {
+    window = BorderlessWindow()
+    window.setContentSize(NSSize(width: 100, height: 100))
```

Comment:
Assuming this is arbitrary?

Reply:
yep

Reply:
we might want to make it customizable at some point